cmd/check.go

```diff
@@ -8,18 +8,19 @@ import (
 
 	"github.com/opdev/opcap/internal/capability"
 	"github.com/opdev/opcap/internal/operator"
+	"github.com/opdev/opcap/internal/logger"
 
 	pkgserverv1 "github.com/operator-framework/operator-lifecycle-manager/pkg/package-server/apis/operators/v1"
 
 	"github.com/spf13/cobra"
 )
 
-// TODO: provide godoc compatible comment for checkCmd
+// TODO: provide godoc compatible comment for checkCmd.
```

Comment:
For one, these should probably just be removed. But they don't appear to be related to this patch. And whatever it is that added a period you should probably turn off. :)